native-to-erc20/oracle/src/events/processInitiateChange/index.js

```diff
@@ -72,7 +72,7 @@ function processInitiateChangeBuilder (config) {
         const foreignBridgeVersion = await foreignBridgeStorage.methods.version().call()
         const message = createNewSetMessage({
           foreignBridgeVersion,
-          newSet: newSet,
+          newSet: [...newSet],
```

Comment:
@Andrew-Pohl that was the fix basically. the `newSet` was immutable because of some web3 internals (it's part of the web3 event)

Reply:
Great spot!! Bet that was a pig to find!